tests/test_process_domain_events.py

```diff
@@ -0,0 +1,126 @@
+import inspect
+import uuid
+
+import pytest
+
+from ddd import DomainEvent
+from ddd import DomainEventCollection
+from ddd import Entity
+from ddd import domain_event_handler
+from ddd import process_domain_events
+from ddd.domain_event_collection import DomainEventDescriptor
+from ddd.domain_event_collection import Locked
+from ddd.domain_event_handler import delete_domain_event_handler
+from ddd.domain_event_handler import get_instance_getter
+from ddd.domain_event_handler import set_instance_getter
+from ddd.process_domain_events import MaximumRecursionException
+
+
+@pytest.fixture
+def empty_handler():
+    class Handler:
+
+        @domain_event_handler
+        def empty_handle(self, domain_event: DomainEvent):
+            pass
+
+    yield Handler
+    delete_domain_event_handler(DomainEvent)
+
+
+class Aggregate(Entity):
+    domain_events = DomainEventDescriptor()
```

Comment:
How about not scaring developers by using a word "Descriptor"? What about just `domain_events = DomainEvents()`? Or do you think it makes the code less clear and straightforward?

Reply:
I think DomainEvents is good
I'll rename